### PR TITLE
Fix build and QA

### PR DESCRIPTION
**Pull request by saberduck**

### CODE REVIEW

File: .cirrus.yml [hunk 1]
```diff
@@ -80,6 +80,9 @@ qa_task:
     <<: *CONTAINER_DEFINITION
     cpu: 4
     memory: 8G
+  node_version_script:
+    - node --version
+    - npm --version
```
Comment (yassin-kammoun-sonarsource) [inline .cirrus.yml:85]: Was this added for debugging purposes or do you intentionally need these?

Comment (saberduck) [reply .cirrus.yml:85]: it's for debugging, but I would keep them, it makes it easier to understand the version of node used

Comment (yassin-kammoun-sonarsource) [reply .cirrus.yml:85]: 👍

**Review by yassin-kammoun-sonarsource — APPROVED** [review]
Thanks for fixing that!

**Pull request merged**